### ISSUE TITLE
Education modules links should point to S3

### ISSUE THREAD

**Issue by joewiz** [eXistSolutions/hsg-shell#59]
On http://localhost:8080/exist/apps/hsg-shell/education/modules/documents-intro, the link to the "Project Index" should point to S3 but instead is being treated as a local link:

    <a href="/exist/apps/hsg-shell/documents.pdf">Project Index (PDF, 821 KB)</a>

On http://history.state.gov/education/modules/documents-intro, this link points to http://s3.amazonaws.com/static.history.state.gov/edu-modules/documents.pdf.

**Comment by dizzzz** [comment]
interesting; I hoped that a reference like $extern would work, but that is pointing a different location....

```
<p>{$availability-message}. Download the <a href="{concat('$extern/', $config:S3_DOMAIN , '/edu-modules/', $file-name)}">{$file-label}</a>.</p>
```

**Comment by dizzzz** [comment]
suggest to rewrite 

'''xquery
replace(
            replace($href, "\$extern", "https://history.state.gov"),
            "\$app",
            $path-to-app
        )
```

with an additional statement to replace `$s2amazon` with `http://s3.amazonaws.com`

will do tomorrow

**Comment by dizzzz** [comment]
as discussed with @joewiz  `http://s3.amazonaws.com/static.history.state.gov/edu-modules/documents.pdf` should be `https://s3.amazonaws.com/static.history.state.gov/edu-modules/documents.pdf`

**Comment by joewiz** [comment]
Fixed by #91.
